pkg/plugins/platform/local.go

```diff
@@ -854,7 +854,19 @@ func (l *LocalApp) Down(removeData bool) error {
 				}
 			}
 		}
-		dir := filepath.Dir(l.AppConfig.DataDir)
+		// Check to see if the AppRoot exists.
+		// If it does not exist, manually set the data directory to remove.
+		var dir string
+		if fileutil.FileExists(l.AppConfig.AppRoot) {
+			dir = filepath.Dir(l.AppConfig.DataDir)
```

Comment:
Another way to deal with this is to make sure that AppConfig.DataDir was already set correctly. I'd like that better. Could we push this farther up, to make sure that AppConfig gets populated a bit better? We're building it, right? So we can build it right? I *think* the question is about why it didn't get set right in the first place.